### PR TITLE
Auto-create releases when a build is tagged. 

### PR DESCRIPTION
**Pull request by vintagepc**
Per title:

- Run builds on tags
- Auto-upload the FW file and create a draft release for review/publication
 - [x] Requires marvinpinto/action-automatic-releases@latest to be whitelisted.

### CODE REVIEW

**Review by 3d-gussner — APPROVED** [review]
CMAKE build creates release folder and copies correct files to be attached to the draft release

**Review by DRracer — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ on:
     - '*'
   push:
     branches: [ MK3, MK3_* ]
-
+    tags:
+    - 'v*'
+
```
Comment (3d-gussner) [inline .github/workflows/build.yml]: Minor thing. white spaces

**Review by 3d-gussner — APPROVED** [review]
Tested on my github reposoitory

- [X] Creates a Draft Release
  - [X] with all `hex` and `zip` files
- [X] Tested the different hex (including the zipped) files on MK404 and all show correct version, build number and hash.

File: .github/workflows/build.yml (hunk 1 above)
Comment (vintagepc) [reply .github/workflows/build.yml]: I'm not sure I follow. Diff shows a blank line removed on the left, it's readded below on the right

**Pull request merged**